### ISSUE TITLE
Adding color options

### ISSUE THREAD

**Issue by ghtmtt** [ghtmtt/DataPlot#3]
It would be very nice to have the chance to customize the plot color..

Using the `QgsColorButtonV2`

[http://geoapis.sourcepole.com/qgispyapi/qgscolorbuttonv2](http://geoapis.sourcepole.com/qgispyapi/qgscolorbuttonv2)

**Comment by ghtmtt** [comment]
It is quite tricky.. some import and conversion between hex and rgb..

```
import matplotlib.colors as colors

# get the color button and the hex raw color code of the selected color
colbutton = self.colorButton
colorhex = colbutton.color().name()

# convert the hex color code to rgb code
colorrgb = colors.hex2color(colorhex)
```

**Comment by ghtmtt** [comment]
Check the color conversion, sometimes it seems not working properly.

Hex code seem right, but wrong color is used in the plot

**Comment by ghtmtt** [comment]
Fixed with 095a2a85934f9d7d6cf83e02c12d662d72afe32e
